[2025-01-24T07:30:13.512+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T07:30:13.518+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_premise_postgres_dag.extract_gz_task scheduled__2025-01-22T00:00:00+00:00 [queued]>
[2025-01-24T07:30:13.521+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_premise_postgres_dag.extract_gz_task scheduled__2025-01-22T00:00:00+00:00 [queued]>
[2025-01-24T07:30:13.521+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-24T07:30:13.525+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_gz_task> on 2025-01-22 00:00:00+00:00
[2025-01-24T07:30:13.528+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=551) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T07:30:13.529+0000] {standard_task_runner.py:72} INFO - Started process 552 to run task
[2025-01-24T07:30:13.529+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_premise_postgres_dag', 'extract_gz_task', 'scheduled__2025-01-22T00:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/Premise Project/ingest_data_to_local_postgres.py', '--cfg-path', '/tmp/tmpdl649kg4']
[2025-01-24T07:30:13.530+0000] {standard_task_runner.py:105} INFO - Job 64: Subtask extract_gz_task
[2025-01-24T07:30:13.546+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_premise_postgres_dag.extract_gz_task scheduled__2025-01-22T00:00:00+00:00 [running]> on host 22ff6bf154fe
[2025-01-24T07:30:13.570+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_premise_postgres_dag' AIRFLOW_CTX_TASK_ID='extract_gz_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-22T00:00:00+00:00'
[2025-01-24T07:30:13.570+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T07:30:13.865+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-24T07:30:13.880+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T07:30:13.880+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_premise_postgres_dag, task_id=extract_gz_task, run_id=scheduled__2025-01-22T00:00:00+00:00, execution_date=20250122T000000, start_date=20250124T073013, end_date=20250124T073013
[2025-01-24T07:30:13.928+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T07:30:13.936+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-24T07:30:13.937+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
